### ISSUE TITLE
(FEATURE REQUEST) Save downloaded files for future use

### ISSUE THREAD

**Issue by Merith-TK** [mondul/HiyaCFW-Helper#71]
As this says, since the helper downloads the files anyways, why not just make a folder `hiya-cfw` and store the files there? they way if its the same nand file, it can just use whats already extracted?

**Comment by mondul** [comment]
Normally, the helper is used once, as the updatable part (TWiLightMenu) can be done without it. I can add a checkbox for that though.

**Comment by Merith-TK** [comment]
Yeah, its just annoying and less efficient because i work off USB drives ALOT, and the constant Writing and Deleting is not only SLOW, it has harmful effects to the USB

**Comment by mondul** [comment]
Added option to keep the downloaded files in v3.6, but it's not what you need. I'm thinking of creating a DSi NAND files extractor based off fatcat and twltool with on-the-fly decryption but that will take some time as I've been busy with work stuff.

**Comment by david-zzj** [comment]
@mondul , Has this function been realized ?

**Comment by Merith-TK** [comment]
It does appear so, a little jank but it does indeed save the files
—Merith
